history/characters/320000_centaur.txt

```diff
@@ -5439,14 +5175,22 @@
 }
 #Dynasty 126005
 320544={
-	name=Vozh
+	name=Kolk
 	dynasty=126005
-	dna="pdmaapckdcl"
+	dna="ppocnlhkcld"
 	culture=centaur religion=theradras_worship
-	martial=7 diplomacy=6 stewardship=5 intrigue=7 learning=7
-	add_trait=midas_touched add_trait=gluttonous add_trait=kind add_trait=arbitrary 
-	1.8.27={ birth=yes }
-	68.5.25={ death=yes }
```

Comment:
Date of death is changed, but in `history` files, he gives titles in 68.5.25, not in 58.4.17 when he died.
![image](https://user-images.githubusercontent.com/46384992/57330497-44983600-7127-11e9-9786-c4df0cc53041.png)